### PR TITLE
Added the option to pass a Spring profile name as argument to the run script

### LINKED ISSUE #113: Run script for development testing should support the usage of spring profiles
As developer, I would like to run the admin application with different profiles to be able to compare several development versions.

Therefore it would be helpful if the run script would allow to pass a spring profile name as parameter.

### PR DESCRIPTION
**Pull request by gr-hovest-atb**
closes #113

### CODE REVIEW

File: run-admin-web-application.sh [hunk 1]
```diff
@@ -2,13 +2,15 @@
 
 # Script to run the CoatRack admin web application 
 
+# Optional: pass a Spring profile name as first command line argument
```
Comment (BrunoSilvaATB) [inline run-admin-web-application.sh:5]: So regarding the extra argument with the profile name. The user might not see these comments, only if he opens it with a notepad or notepad++ for example,
Therefore, my question is should we also add this to the readme?

**Comment by BrunoSilvaATB** [comment]
Regarding funtionality, the script is working well.
I am on Windows and I used the Git Bash to run it.

File: run-admin-web-application.sh (hunk 1 above)
Comment (gr-hovest-atb) [reply run-admin-web-application.sh:5]: In general I agree, however right now only CoatRack developers can create a custom profile, so right now it is not yet really relevant for others.

Comment (gr-hovest-atb) [reply run-admin-web-application.sh:5]: I added it to the README.

Comment (BrunoSilvaATB) [reply run-admin-web-application.sh:5]: I checked and it looks very nice. So I consider this conversation solved

**Review by BrunoSilvaATB — APPROVED** [review]
The code is well structured, the instructions to use it are documented and the functionality is working well. So I approve this PR

**Review by Monezz — APPROVED** [review]

**Pull request merged**